src/OrbitGl/FrameTrack.h

```diff
@@ -74,7 +73,8 @@ class FrameTrack : public TimerTrack {
   [[nodiscard]] float GetMaximumBoxHeight() const;
   [[nodiscard]] float GetAverageBoxHeight() const;
 
-  orbit_grpc_protos::InstrumentedFunction function_;
+  uint64_t function_id_;
+  orbit_client_data::FunctionInfo function_;
```

Comment:
BTW, I am also wondering why function_id isn't just a field of FunctionInfo. That would be a nice follow up PR.

Reply:
Agreed (I actually asked about it in the chat because FindFunctionIdSlow seemed quite intentional about not putting the id in it)

Reply:
I guess, see answer in chat.